implementation/src/main/java/io/smallrye/mutiny/operators/multi/MultiOnFailureInvoke.java

```diff
@@ -0,0 +1,53 @@
+package io.smallrye.mutiny.operators.multi;
+
+import static io.smallrye.mutiny.helpers.ParameterValidation.nonNull;
+
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+
+import org.reactivestreams.Subscription;
+
+import io.smallrye.mutiny.CompositeException;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.helpers.Subscriptions;
+import io.smallrye.mutiny.subscription.MultiSubscriber;
+
+public class MultiOnFailureInvoke<T> extends AbstractMultiOperator<T, T> {
```

Comment:
Make sure with have the associated TCK.

Reply:
We already had `MultiOnFailureInvokeTckTest`